avm/utilities/e2e-template-assets/scripts/Get-HostPoolRegistrationKey.ps1

```diff
@@ -0,0 +1,50 @@
+<#
+.SYNOPSIS
+Returns a registration token to a host pool that is valid for 24h.
+
+.DESCRIPTION
+Returns a registration token to a host pool that is valid for 24h.
+
+.PARAMETER HostPoolName
+Mandatory. The name of the host pool.
+
+.PARAMETER HostPoolResourceGroupName
+Mandatory. The name of the resource group fo the host pool.
+
+.PARAMETER SubscriptionId
+Mandatory. The subscription ID wher the host pool resides.
+
+.EXAMPLE
+./Get-HostPoolRegistrationKey.ps1 -HostPoolName 'hp-01' -HostPoolResourceGroupName 'rg-01' -SubscriptionId '00000000-0000-0000-0000-000000000000'
+
+Output will be '<tokenValue>'.
+#>
+param(
+  [Parameter(Mandatory = $true)]
+  [string] $HostPoolName,
+
+  [Parameter(Mandatory = $true)]
+  [string] $HostPoolResourceGroupName,
+
+  [Parameter(Mandatory = $true)]
+  [string] $SubscriptionId
+)
+
+if ($installedModules.Name -notcontains "Az.DesktopVirtualization") {
```

Comment:
Where is `$installedModules` coming from? It appears the variable has not been initialized

Reply:
fixed